packages/react-interactions/events/src/dom/__tests__/Drag-test.internal.js

```diff
@@ -81,10 +81,8 @@ describe('Drag event responder', () => {
     mouseUpEvent.initEvent('mouseup', true, true);
     divRef.current.dispatchEvent(mouseUpEvent);
 
-    expect(events).toHaveLength(2);
-    expect(events).toEqual(
-      expect.arrayContaining([expect.objectContaining({isChanged: true})]),
-    );
+    expect(events).toHaveLength(0);
```

Comment:
@trueadm The tests in this file look suspicious. As far as I can tell they changed due to JSDOM supporting passive events. At least I noticed that it bails out at https://github.com/eps1lon/react/blob/118d91b1eced121a38cc7da988194f713ca6248a/packages/react-interactions/events/src/dom/Drag.js#L169. Don't know if this was the case before though.